ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1881,6 +1881,23 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetal-upi-ovn-ipv4-p1-f14
+  cluster: build05
```

Comment:
Most of the cases, we do not need to specify `cluster` in configs

Reply:
This is to test baremetal in the rdu2 lab, only reachable from that cluster